### PR TITLE
Add FreeDesktop Portal Notifications

### PR DESCRIPTION
**Pull request by affederaffe**
Implements Linux notifications using the xdg-desktop-portal

### CODE REVIEW

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-  <ItemGroup>
-    <None Remove="Windows\WinRT\windows.ui.notifications.mcidl" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <AdditionalFiles Include="Windows\WinRT\windows.ui.notifications.mcidl" />
-  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
+    <PackageReference Include="Tmds.DBus.Protocol" Version="0.19.0" />
```
Comment (kekekeks) [inline src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj:13]: Those version of tmdbus are known to crash KDE and shouldn't be used with Avalonia codebase

**Review by kekekeks — CHANGES_REQUESTED** [review]

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj (hunk 1 above)
Comment (maxkatz6) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj:13]: @kekekeks @jmacato was it reported anywhere?

Comment (jmacato) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj:13]: https://github.com/AvaloniaUI/Avalonia/pull/16129#issuecomment-2198590303

Comment (maxkatz6) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj:13]: Yeah, it's not an actionable report.

Comment (kekekeks) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.csproj:13]: https://github.com/AvaloniaUI/Avalonia/issues/15655

**Review by jmacato — APPROVED** [review]
Seems to work on Fedora :) nice job y'all

**Pull request merged**